psuedo geospatial work

yulgit1 opened the pull request.

eliotjordan commented on app/models/concerns/geospatial_behavior.rb:
```diff
@@ -0,0 +1,41 @@
+   #allow members to be of any ActiveFedora::Base
+   aggregates :members, predicate: Vocab::PCDMTerms.hasMember,
+                         class_name: 'ActiveFedora::Base',
+                         type_validator: type_validator
```
I don't think ```aggregates``` is necessary here, as this is already defined in [Hydya::PCDM](https://github.com/projecthydra-labs/hydra-pcdm/blob/bce6accb14aba03df5210321d2e52b5008658c72/lib/hydra/pcdm/models/concerns/pcdm_behavior.rb#L5-L7).

yulgit1 commented:
Removed unneeded aggregates, added type and, isType?, and filters_association to works level(geospatial,image,raster,vector), added GeoVocab

todo: spec of works associations, if ok implement at collection and file level

drh-stanford commented on app/models/concerns/external_metadata_file_behavior.rb:
```diff
@@ -0,0 +1,46 @@
+  def concerns_external_metadata_file?
+    true
+  end 
+  def vector
```
I think this isn't needed here...

yulgit1 replied:
will remove

drh-stanford commented:
I don't think the prefix `concerns_` for the condition parameters is part of the pattern

yulgit1 commented:
does it matter? it's just a variable name, as long as the symbol in the condition matches a method in the classes

drh-stanford commented:
we want to follow the hydra works naming/filtering pattern, as shown here:

https://github.com/projecthydra-labs/hydra-works/blob/master/lib/hydra/works/models/concerns/work_behavior.rb#L25

drh-stanford commented:
closing since PR #23 does this now
